quickstart-python/README.md

```diff
@@ -0,0 +1,86 @@
+# Monday-Code Python Quickstart
+
+This Python Flask application is designed to exemplify the practical use of the Monday Code platform by integrating various Monday app functionalities.
+It's an essential resource for developers looking to leverage Monday Code features through a Python environment.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing
+purposes.
+
+### Prerequisites
+
+You need to have Python and pip installed on your machine. You can download Python
```

Comment:
I think venv is a better practice, so I'd point to a good manual how to add venv to the project and activate it